Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2021"
 members = [
   "common",        # everything else, until it's moved to another crate
   "crypto",        # cryptographic primitives and their interfaces
+  "consensus",     # Consensus related logic.
```

Comment:
I like your attention to the details! Though `storage/core` and `storage/inmemory` descriptions also start with a capital letter. 🙃 Personally I prefer to use capital letters everywhere.

Reply:
Since this is a pretty minor change, I suggest @stanislav-tkach to change all the comments here in accordance with his aesthetic sensibilities as a part of this PR (or a separate one if preferred).